tfhe/src/shortint/parameters/mod.rs

```diff
@@ -823,3 +823,9 @@ pub const PARAM_SMALL_MESSAGE_1_CARRY_1: ClassicPBSParameters = PARAM_MESSAGE_1_
 pub const PARAM_SMALL_MESSAGE_2_CARRY_2: ClassicPBSParameters = PARAM_MESSAGE_2_CARRY_2_PBS_KS;
 pub const PARAM_SMALL_MESSAGE_3_CARRY_3: ClassicPBSParameters = PARAM_MESSAGE_3_CARRY_3_PBS_KS;
 pub const PARAM_SMALL_MESSAGE_4_CARRY_4: ClassicPBSParameters = PARAM_MESSAGE_4_CARRY_4_PBS_KS;
+
+pub const PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM: ClassicPBSParameters =
+    PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64;
+
+pub const PARAM_MESSAGE_2_CARRY_2_TUNIFORM: ClassicPBSParameters =
+    PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM;
```

Comment:
I don't think we need the aliases

Reply:
Ok.

Reply:
nevermind the params are still accesible from the full path